language/pl/system.lang

```diff
@@ -201,6 +201,10 @@ $_Lang['LanguagesAvailable']    = [
         'flag_emoji'            => '🇵🇱',
         'name'                  => 'Polski'
     ],
+    'fr'                        => [
```

Comment:
Same as in other places, please remove this from your Pull Request for now. These lines should be brought back once the entire translation is complete.